src/depackers/miniz.h

```diff
@@ -303,6 +304,10 @@ typedef int mz_bool;
 #define MZ_FALSE (0)
 #define MZ_TRUE (1)
 
+/* tinfl doesn't have a nicer way of limiting the output buffer size. */
```

Comment:
We can move these either down after the commented-out originals or above among the libxmp-specific config section.

Reply:
I misread and thought it was in the libxmp hacks, sorry. I moved it next to the old `MZ_REALLOC`/etc defines.